public/Startup.html

```diff
@@ -115,20 +186,14 @@ <h2 class="h">Lets start dating!</h2>
                     </label>
                 </p>
                 <p>
-                    <label>My location:
```

Comment:
There is a dangling </p> tag ( line 182 )

Reply:
Its not dangling, its the closing /p for the tag before the label